Development/v4.x/backend/configuration_oidc.md

```diff
@@ -1,169 +1,33 @@
-
-```javascript
-
-module.exports.sampleLoginCallback = function(req, done) {
-  return function(err, user, identity, token) {
-
-    var authInfo = {
-      identity: identity,
-    };
-    if (token) {
-      authInfo.accessToken = token;
-    }
-
-    const requestURL = getUserURL(identity);
-    if (!requestURL){
-      logger.logError(`unexpected authenticator type: ${identity.provider}`);
-      done();
-      return;
-    }
-    request(requestURL, function (error, response, body) {
-      // ask external system for the user's profile information
-      if (error){
-        // and error occurred talking to the external system (possibly from a connection issue)...deny login
-        logger.logError(`error talking to external ${error.message}`);
-        done(err, null, null);
-        return;
-      }
-      if (response.statusCode == 200) {
-        const bodyObj = JSON.parse(body);
-        logger.logInfo("user service returned", bodyObj);
-      }
-      else{
-        logger.logError(`external system replied with an error ${response.statusCode} - ${body}.`);
-        // external system replied with a negative response...deny login
-        done(err, null, null);
-        return;
-      }
-      //accept login
-      done(err, user, authInfo);
-    });
-  };
-};
+The `frontend.config.json` requires the following variables to be set to work with OIDC.
 ```
-### Add information to User's Profile 
+ 
+      "accessTokenPrefix": "Bearer ", # make sure there is a whitespce between Bearer and the end quotes
+      "lbBaseURL": "https://scicat.myinstitute.org" # the address you are returning to after authentication if it is not 127.0.0.1:3000
+      "oAuth2Endpoints": [
+       {"displayText": "My Institute SSO", "authURL": "api/v3/auth/oidc"}
+      ]
 
-By default profile information from the Authentication Provider will be added to the user's profile. This profile is stored in the DaCat database `UserIdentity` collection. This collection is created by the loopback-component-passport library, and is also used for LDAP authentication. The backend LDAP integrations populate this collection with the user's LDAP profile information automatically with a callback. One of the most important things that is added to the profile is the user's accessGroups field, which is critical to calculating a user's privilege in SciCat.
+```
+SciCat requires you to specify the `accesstokenprefix` in order to find the the token in the OAuth response. 
 
-One way to mimic this with an OAuth2 provider is to use a `before save` [Loopback Operation Hook](https://loopback.io/doc/en/lb3/Operation-hooks.html). This hook will get called as the user logs in and allows you to add information to the `UserIdentity` collection, which will be used in a variety of places, including access controls.
+The OIDC mechanism assues that you are returning to 127.0.0.1:3000, if your requiret to return to a different address or port you need to edid the `lbBaseURL`  variable. 
```

Comment:
What do you think about this description of the `lblBaseURL`:

"`lblBaseURL` configures the base URL that the Scicat frontend to Backend (API) service that the frontend will communicate with. It will therefore match the roots of the `OIDC_CALLBACK_URL` and the `OIDC_SUCCESS_URL` in the backend OIDC configurations.

Reply:
Thanks for the review Dylan. I have added this sentence as it describes it better.